TeachingRecordSystem/src/TeachingRecordSystem.Core/Events/PersonDetailsUpdatedEvent.cs

```diff
@@ -22,5 +24,6 @@ public enum PersonDetailsUpdatedEventChanges
     EmailAddress = 1 << 4,
     MobileNumber = 1 << 5,
     NationalInsuranceNumber = 1 << 6,
-    AnyNameChange = FirstName | MiddleName | LastName
+    NameChange = FirstName | MiddleName | LastName,
+    OtherDetailsChange = DateOfBirth | EmailAddress | MobileNumber | NationalInsuranceNumber
```

Comment:
Might not be completely clear that `Other` means 'other than name'..
```suggestion
    OtherThanNameChange = DateOfBirth | EmailAddress | MobileNumber | NationalInsuranceNumber
```